kie-spring/src/test/java/org/kie/spring/CodeVersion.java

```diff
@@ -18,6 +18,6 @@
 public class CodeVersion {
 
     // Do not modify this: this is automatically incremented by the maven replacer plugin
-    public final static String VERSION = "7.0.0-SNAPSHOT";
+    public final static String VERSION = "7.1.0-SNAPSHOT";
```

Comment:
I am not sure why we need this in the code as it causes issues every single time we are updating versions. The value needs to (should?) match the version in the POMs. However, we should take a look at removing this completely.